src/main/java/com/synacy/gradprogram/spock/exercise/DateUtils.java

```diff
@@ -8,4 +10,14 @@ public Date getCurrentDate() {
     return new Date();
   }
 
+  public boolean isDateWithinRefundPeriod(Date givenDate) {
```

Comment:
The DateUtils class is supposed to be generic, adding the `DAYS_WITHIN_REFUND_PERIOD` in here is adding a business logic (the concept of refund period) and it's not good. You can improve the method by accepting "refund period" instead.

Also this method can be improved :+1: